lib/tasks/scheduler.rake

```diff
@@ -1,7 +1,7 @@
 require 'happy_hood/slack/client'
 
 desc 'Collect Zillow Zestimate for each House'
-task :house_valuation_collector => :environment do
+task house_valuation_collector: :environment do
```

Comment:
These two are functionally the same, except the first looks like Ruby out of the late 2000s.

Reply:
Lol rude
jk I just copied some example from somewhere